pilot/pkg/autoregistration/controller.go

```diff
@@ -72,17 +73,6 @@ var (
 )
 
 const (
-	// TODO use status or another proper API instead of annotations
-
-	// AutoRegistrationGroupAnnotation on a WorkloadEntry stores the associated WorkloadGroup.
-	AutoRegistrationGroupAnnotation = "istio.io/autoRegistrationGroup"
```

Comment:
May be it is lot simpler to replace the name at this constant and use the constants every where?

Reply:
Yes, but just align with others